packages/android_alarm_manager_plus/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: android_alarm_manager_plus
 description: Flutter plugin for accessing the Android AlarmManager service, and
   running Dart code in the background when alarms fire.
-version: 1.3.0
+version: 1.2.0
```

Comment:
@mhadaily I think this is an unnecessary change. And there are other similar changes like this.

Reply:
You are absolutely right. @adityathakurxd  I hope you will take care of it.

Reply:
Yes, I am sorry about this. I'll take care of it and make it properly.